src/Markup/Avalonia.Markup.Xaml/Converters/TimeSpanTypeConverter.cs

```diff
@@ -15,7 +21,7 @@ public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceT
         public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
         {
             var valueStr = (string)value;
-            if (!valueStr.Contains(":"))
+            if (!valueStr.Contains(Colon))
```

Comment:
@workgroupengineering I think it would be cleaner to replace all of it with one helper internal function:
```
#if !NET6_0_OR_GREATER
internal static bool Contains(string str, char search)
{
    return str.Contains(search.ToString());
}
#endif
```

Reply:
Not sure though if this helper will be visible in every place where we need it.